docs/using-airbyte/getting-started/oss-quickstart.md

```diff
@@ -315,11 +323,11 @@ curl -LsfS https://get.airbyte.com | bash -
 ```
 
 6. Run the `abctl` command and install Airbyte:
-:::tip
-By default, `abctl` only configures an ingress rule for the host `localhost`. In order to ensure that Airbyte can be accessed outside of the EC2 instance, you will need to specify the `--host` flag to the `local install` command, providing the FQDN of the host which is hosting Airbyte. For example, `abctl local install --host airbyte.company.example`.
-:::
-:::tip
-By default, `abctl` will listen on port 8000. If port 8000 is already in used or you require a different port, you can specify this by passing the `--port` flag to the `local install` command. For example, `abctl local install --port 6598`
+   :::tip
```

Comment:
It's weird to me that there are two tips in fast follow to each toher. Can we make one of these a note or just change it to normal text?